### PR TITLE
Fix issue with attributes

### PR DESCRIPTION
**Pull request by harture**
Fix for CLOUDTRUST-1061

### CODE REVIEW

File: pkg/management/component.go [hunk 1]
```diff
@@ -272,6 +272,23 @@ func (c *component) UpdateUser(ctx context.Context, realmName, userID string, us
 
 	userRep = api.ConvertToKCUser(user)
 
+	// Merge attributes coming from old user and updated one to not loose anything
```
Comment (bsoniam) [inline pkg/management/component.go]: Merge the attributes coming from the old user representation and the updated user representation in order not to lose anything

Comment (bsoniam) [reply pkg/management/component.go]: This is a suggestion as your comment is a bit fuzzy

Comment (bsoniam) [reply pkg/management/component.go]: Maybe you can also add tests with having empty attributes in the updated user representation ?

Comment (harture) [reply pkg/management/component.go]: Thanks for the improved comments.
Good idea for the unittests, I did it.
